### PR TITLE
Unify Design - Purple Theme

### PR DESCRIPTION
**Pull request by razakhawaja**
Adds purple theme + some personal customizations.

### CODE REVIEW

File: pages/about.tsx [hunk 1]
```diff
@@ -5,17 +5,11 @@ const Home = () => {
...
-        <meta
-          property="og:description"
-          content="This will be the description of the website"
-        />
-        <link rel="icon" href="/favicon.ico" />
+        <title>Raza Khawaja</title>
+        <meta name="description" content="Raza Khawaja's Portfolio" />
+        <meta property="og:title" content="Raza Khawaja" />
+        <meta property="og:description" content="Raza Khawaja's Portfolio" />
+        <link rel="icon" href="/R.svg" />
```
Comment (hamza1220) [inline pages/about.tsx:12]: Don't use a full size icon as a favicon. use a favicon generator to make your favicon.

Comment (hamza1220) [reply pages/about.tsx:12]: Will have to revert this change in all files that were edited

File: containers/About/index.tsx [hunk 2]
```diff
@@ -16,44 +18,65 @@ const About = () => {
...
+              style={{ color: "#8E6F3E" }}
+            >
+              Purdue
+            </span>{" "}
+            studying UX Design. Most recently, I was a full-stack software
+            engineer at
+            <Link href="https://www.educative.io">
+              <span
+                className="cursor-pointer font-semibold hover:opacity-75"
+                style={{ color: "#5553FF" }}
```
Comment (hamza1220) [inline containers/About/index.tsx:35]: use the gradient classes `gradient-purple text-gradient` for this instead

File: containers/About/index.tsx [hunk 3]
```diff
@@ -16,44 +18,65 @@ const About = () => {
...
-        <div className="sm:col-start-1 sm:col-end-4 leading-normal text-lg text-blueGray-500 dark:text-gray-600">
-          <p className="mb-2">
-            Hello! My name is Hamza and I enjoy creating things that live on the
-            internet. My interest in web development started back in Lorem ipsum
-            dolor sit amet, consectetur adipiscing elit. Mauris arcu ipsum,
-            rhoncus sed gravida et, luctus eu enim. Donec a nunc placerat velit
-            vestibulum porttitor sit
+        <div className="sm:col-start-1 sm:col-end-4 leading-normal text-xl text-blueGray-500 dark:text-gray-600">
+          <p className="mb-4">
+            I'm currently a student at{" "}
```
Comment (hamza1220) [inline containers/About/index.tsx:23]: all of this along with the jumbotron info should be dynamic. Can leave here for now but will need to shift later

File: pages/index.tsx [hunk 4]
```diff
@@ -24,17 +24,17 @@ const Home = ({ works }: InferGetStaticPropsType<typeof getStaticProps>) => {
   return (
     <>
       <Head>
-        <title>S. Hamza Ahmad</title>
+        <title>Raza Khawaja</title>
```
Comment (hamza1220) [inline pages/index.tsx:27]: We need to make the metadata dynamic too

**Review by hamza1220 — COMMENTED** [review]
Change the resume download button's color on hover/focus/active

File: containers/About/index.tsx (hunk 3 above)
Comment (razakhawaja) [reply containers/About/index.tsx:23]: I'll create an issue.

**Pull request merged**